### PR TITLE
Upstream Net-DRI-0.96_10

### PR DESCRIPTION
**Pull request by thedarkwinter**

### CODE REVIEW

**Comment by thedarkwinter** [comment]
`
t/Net/DRI/Protocol/EPP/Core.t                                    (Wstat: 256 Tests: 276 Failed: 1)
  Failed test:  273
  Non-zero exit status: 1
t/Net/DRI/Protocol/EPP/Extensions/Afilias/IDN.t                  (Wstat: 768 Tests: 10 Failed: 3)
  Failed tests:  5, 7, 9
  Non-zero exit status: 3
t/Net/DRI/Protocol/EPP/Extensions/Afilias/Restore.t              (Wstat: 512 Tests: 0 Failed: 0)
  Non-zero exit status: 2
  Parse errors: Bad plan.  You planned 3 tests but ran 0.
t/Net/DRI/Protocol/EPP/Extensions/FCCN.t                         (Wstat: 256 Tests: 24 Failed: 1)
  Failed test:  13
  Non-zero exit status: 1
t/Net/DRI/Protocol/EPP/Extensions/Fee/Fee-1.0.t                  (Wstat: 65280 Tests: 5 Failed: 2)
  Failed tests:  1-2
  Non-zero exit status: 255
  Parse errors: Bad plan.  You planned 146 tests but ran 5.
t/Net/DRI/Protocol/EPP/Extensions/NeuLevel/IDN.t                 (Wstat: 512 Tests: 3 Failed: 2)
  Failed tests:  1, 3
  Non-zero exit status: 2
t/Net/DRI/Protocol/EPP/Extensions/Tango.t                        (Wstat: 256 Tests: 97 Failed: 1)
  Failed test:  2
  Non-zero exit status: 1
t/Net/DRI/Protocol/EPP/Extensions/UniRegistry.t                  (Wstat: 512 Tests: 55 Failed: 2)
  Failed tests:  35, 49
  Non-zero exit status: 2
t/Net/DRI/Protocol/EPP/Extensions/VeriSign/DefReg.t              (Wstat: 256 Tests: 66 Failed: 1)
  Failed test:  57
  Non-zero exit status: 1
`

**Comment by thedarkwinter** [comment]
The following were omitted as the local versions are maintained separately.

CentralNic::AuxContact
CentralNic::RegType
IT
IT::Contact
IT::Domain
IT::Notifications	
PL
PL::Contact
PL::Domain

The following updates were causing problems and have been ignored
Afilias::IDNLanguage.pm
Afilias::Restore.pm
NeuLevel::IDNLanguage.pm
VeriSign::IDNLanguage.pm

File: lib/Net/DRI/Protocol/EPP/Extensions/Nomulus.pm
Comment (thedarkwinter) [inline lib/Net/DRI/Protocol/EPP/Extensions/Nomulus.pm:1]: Should we link CRR and Nomulus modules? I think they are the same thing?

File: t/Net/DRI/Protocol/EPP/Core.t
Comment (thedarkwinter) [inline t/Net/DRI/Protocol/EPP/Core.t:554]: I feel like `only_local_extensions` has been upset by the way some extensions run setup (see sub setup in VeriSign::Sync). Its not critical, but it required that these extensions are added to the test.

File: t/Net/DRI/Protocol/EPP/Extensions/FCCN.t
Comment (thedarkwinter) [inline t/Net/DRI/Protocol/EPP/Extensions/FCCN.t:75]: This looks like a fix (extension change) (and see the module itself). I have therefor accepted the change, even though I don't know for sure. I think there is some unmerged work on .pt somewhere too.

File: t/Net/DRI/Protocol/EPP/Extensions/UniRegistry.t
Comment (thedarkwinter) [inline t/Net/DRI/Protocol/EPP/Extensions/UniRegistry.t:186]: see note elsewhere, a change to VeriSign::Sync module makes it automatically load even when local_only_extensions. Not a biggue

**Review by thedarkwinter — COMMENTED** [review]
I am happy with all of these changes. Will get another set of eyes on it!

File: EPP_namespaces.txt
Comment (paullojorgge) [inline EPP_namespaces.txt:1]: Only includes the namespaces from original PM Net-DRI-0.96_10 version correct?

File: eg/whois.pl
Comment (paullojorgge) [inline eg/whois.pl:88]: One of this annoying things... such mentioning it

File: lib/Net/DRI/Protocol/EPP/Extensions/FCCN/Domain.pm
Comment (paullojorgge) [inline lib/Net/DRI/Protocol/EPP/Extensions/FCCN/Domain.pm:69]: whitespace

File: lib/Net/DRI/Protocol/EPP/Extensions/Nomulus.pm
Comment (paullojorgge) [reply lib/Net/DRI/Protocol/EPP/Extensions/Nomulus.pm:1]: When I merged all versions I simply kept them seperate just avoid possible problems and/or lose functionalities.

- https://github.com/paullojorgge/Net-DRI/tree/pm-Net-DRI-0.96_13/lib/Net/DRI/Protocol/EPP/Extensions/Nomulus

But definitely something that we need to review as soon as all versions are merged :)

File: t/Net/DRI/Protocol/EPP/Extensions/UniRegistry.t
Comment (paullojorgge) [reply t/Net/DRI/Protocol/EPP/Extensions/UniRegistry.t:186]: Should be fine for now but when all PM versions are merged we should double check and test properly those `only_local_extensions`: `Core.t` and `UniRegistry.t`

**Review by paullojorgge — APPROVED** [review]
Some minor comments. We should be ready to move to Net-DRI-0.96_11.
As soon as everything is merged it could be interesting to diff against https://github.com/paullojorgge/Net-DRI/tree/pm-Net-DRI-0.96_13 just to see how bad/different it looks like :sweat_smile:

File: EPP_namespaces.txt
Comment (thedarkwinter) [reply EPP_namespaces.txt:1]: yes, will update for future versions and check our namespaces when all is complete.

File: eg/whois.pl
Comment (thedarkwinter) [reply eg/whois.pl:88]: yes, normally i would prefer trim, but trying to keep aligned to upstream :P

**Comment by thedarkwinter** [comment]
thanks @paullojorgge :)

**Pull request merged**
